_includes/footer.html

```diff
@@ -27,30 +27,4 @@
       </div>
     </div>
   </div>
-</footer>
-
-<!-- Digital Analytics Program roll-up, see https://analytics.usa.gov for data -->
-<script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency={{site.dap_agency}}"></script>
-
-<!-- Google Analytics -->
-<script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-  ga('create', '{{site.google_analytics_ua}}', '{{site.url}}');
-
-  // anonymize user IPs (chops off the last IP triplet)
-  ga('set', 'anonymizeIp', true);
-
-  // forces SSL even if the page were somehow loaded over http://
-  ga('set', 'forceSSL', true);
-
-  ga('send', 'pageview');
-</script>
-
-<script src="{{site.baseurl}}/assets/uswds/js/uswds.min.js"></script>
-{% asset index.js %}
-
-</body>
-</html>
```

Comment:
The scripts were moved to a separate partial specifically for them. In addition, as part of a cleanup, the unmatched `body` and `html` closing tags were moved to where the opening tags live.